jobs/e2e-test-cypress.groovy

```diff
@@ -0,0 +1,138 @@
+// Single job for cypress e2e tests.
+// 
+// cypress e2e tests are the smoketests run in the webapp/feature/cypress repo, 
+// that hit a live website using lambdatest cli.
+//
+// TODO(ruslan): integrate later this job to the e2e tests pipeline.
+// This job runs lambdatest-cypress-cli in the webapp environment 
+// depending on how parameters are specified.
+// 
+
+
+@Library("kautils")
+// Classes we use, under jenkins-jobs/src/.
+import org.khanacademy.Setup;
+// Vars we use, under jenkins-jobs/vars/.  This is just for documentation.
+//import vars.exec
+//import vars.notify
+//import vars.withTimeout
+//import vars.withSecrets
+
+public class TestFailed extends Exception {}
+
+new Setup(steps
+
+).addStringParam(
+   "GIT_REVISION",
+   """The name of a webapp branch to use when building.""",
+   "master"
+
+).addStringParam(
+   "URL",
+   "The url-base to run these tests against.",
+   "https://www.khanacademy.org/"
+
+).addStringParam(
+   "SLACK_CHANNEL",
+   "The slack channel to which to send failure alerts.",
+   "#cypress-testing"
+
+).addStringParam(
+   "DEPLOYER_USERNAME",
+   """Who asked to run this job, used to ping on slack.
+Typically not set manually, but rather by other jobs that call this one.""",
+   "username"
+
+).addStringParam(
+   "REVISION_DESCRIPTION",
+   """Set by the buildmaster to give a more human-readable description
+of the GIT_REVISION, especially if it is a commit rather than a branch.
+Defaults to GIT_REVISION.""",
+   ""
+
+).addStringParam(
+   "NUM_WORKER_MACHINES",
+   """How many worker machines to use. Max available is 20.""",
+   "20"
+
+).addStringParam(
+   "TEST_RETRIES",
+   """How many retry attempts to use. By default is 1.""",
+   "1"
+
+).addChoiceParam(
+   "TEST_TYPE",
```

Comment:
This is not used.  Did you want to take it out?  Or add a TODO somewhere to use it.